landscape/README.md

```diff
@@ -0,0 +1,36 @@
+## Goal
+
+The [SAFE roadmap](../roadmap.md) includes describing the landscape of
+cloud-native security. We evaluated categories in the CNCF Landscape and
+determined the need for a [modified approach](cncf.md).
+
+We propose major categories that are similar to those in the
+[CNCF Landscape](https://landscape.cncf.io/), but with sub-categories that
+highlight the main security considerations in each category.
+
+We propose [categories](categories.md) as a draft structure for a “Cloud Native
+Security Landscape”. We drafted this document after reviewing the current list
+of projects in the CNCF Landscape and recommendations by SANS and Gartner for
+good security practices, as well as drawing on the experience of the SAFE WG.
+
+## Landscape Process
+
+- [X] Draft [categories](categories.md)
+- [ ] Determine approach to category mapping (see below)
+- [ ] Map cloud-native tools into categories (adjusting categories as needed)
+- [ ] Validate categories and landscape with review by makers and users of
+    cloud-native security solutions, as well as partner working groups
+
+A note on how the work of mapping tools into the sub-categories may proceed:
```

Comment:
Maybe make this into a subsection, that we can then link above (where it says "see below")